mirgecom/mpi.py

```diff
@@ -121,6 +121,20 @@ def wrapped_func(*args, **kwargs):
             raise RuntimeError("mpi4py.MPI imported before designated MPI entry "
                         "point. Check for prior imports.")
 
+        actx_class = kwargs.pop("actx_class", None)
+        if actx_class:
```

Comment:
I'm not sure about this being mixed in with the MPI functionality. I can see *why*, but I'm not sure it's a good idea.